src/main/java/org/jenkinsci/infra/tools/HyperLocalPluginManager.java

```diff
@@ -400,21 +415,23 @@ public <T> List<T> findComponents(Class<T> type) {
     }
 
     /**
-     * This is pretty much a copy of the final ExtensionFinder.Sezpoz class from 1.651 fitted
+     * This is pretty much a copy of the final ExtensionFinder.Sezpoz class from
+     * 1.651 fitted
```

Comment:
Note: something we might need to do in this repo is do a refresh on Sezpoz.  A good ticket for this repository.

Reply:
Sure, would like to know more about this in the office hours.